prowler/providers/aws/lib/quick_inventory/quick_inventory.py

```diff
@@ -285,23 +284,32 @@ def create_output(resources: list, provider: AwsProvider, args):
     print(f" - JSON: {args.output_directory}/{output_file + json_file_suffix}")
 
     # Send output to S3 if needed (-B / -D)
-    for mode in ["csv"]:
-        if args.output_bucket or args.output_bucket_no_assume:
-            # Check if -B was input
-            if args.output_bucket:
-                output_bucket = args.output_bucket
-                bucket_session = provider.session.current_session
-            # Check if -D was input
-            elif args.output_bucket_no_assume:
-                output_bucket = args.output_bucket_no_assume
-                bucket_session = provider.original_session
-            send_to_s3_bucket(
-                output_file,
-                args.output_directory,
-                mode,
-                output_bucket,
-                bucket_session,
-            )
+
+    if args.output_bucket or args.output_bucket_no_assume:
+        # Check if -B was input
+        if args.output_bucket:
+            args.output_bucket
+            bucket_session = provider.session.current_session
+        # Check if -D was input
+        elif args.output_bucket_no_assume:
+            args.output_bucket_no_assume
+            bucket_session = provider.original_session
+
+        bucket_session.client("s3")
+
+        # TODO: fix the above
+        # CSV
+        # s3_client.upload_file(
+        #     {args.output_directory} / {output_file + csv_file_suffix},
+        #     output_bucket,
+        #     object_name,
+        # )
+        # JSON
+        # s3_client.upload_file(
+        #     {args.output_directory} / {output_file + csv_file_suffix},
+        #     output_bucket,
+        #     object_name,
+        # )
```

Comment:
@sergargar could you please fix this. We should not use the S3 class anymore here since the QuickInventory has not an output created yet.

Reply:
Done!